public/wp-content/themes/newsup/index-banner.php

```diff
@@ -5,23 +5,23 @@
   $newsup_background_image = get_header_image();
 }
 ?>
-<div class="mg-breadcrumb-section" style='background: url("<?php echo esc_url( $newsup_background_image ); ?>" ) repeat scroll center 0 #143745;'>
-<?php $newsup_remove_header_image_overlay = get_theme_mods('remove_header_image_overlay',true);
-if($newsup_remove_header_image_overlay == true){ ?>
-  <div class="overlay">
-<?php } ?>
-    <div class="container-fluid">
-      <div class="row">
-        <div class="col-md-12 col-sm-12">
-			    <div class="mg-breadcrumb-title">
-            <h1><?php the_title(); ?></h1>
-          </div>
-        </div>
-      </div>
-    </div>
-  <?php $newsup_remove_header_image_overlay = get_theme_mods('remove_header_image_overlay',true);
-if($newsup_remove_header_image_overlay == true){ ?>
-  </div>
-<?php } ?>
-</div>
-<div class="clearfix"></div>
\ No newline at end of file
+<!--<div class="mg-breadcrumb-section" style='background: url( "--><?php //echo esc_url( $newsup_background_image ); ?>
+<?php //$newsup_remove_header_image_overlay = get_theme_mods('remove_header_image_overlay',true);
+//if($newsup_remove_header_image_overlay == true){ ?>
+<!--  <div class="overlay">-->
+<?php //} ?>
+<!--    <div class="container-fluid">-->
+<!--      <div class="row">-->
+<!--        <div class="col-md-12 col-sm-12">-->
+<!--			    <div class="mg-breadcrumb-title">-->
+<!--            <h1>--><?php //the_title(); ?><!--</h1>-->
+<!--          </div>-->
+<!--        </div>-->
+<!--      </div>-->
+<!--    </div>-->
+<!--  --><?php //$newsup_remove_header_image_overlay = get_theme_mods('remove_header_image_overlay',true);
+//if($newsup_remove_header_image_overlay == true){ ?>
+<!--  </div>-->
+<?php //} ?>
+<!--</div>-->
+<!--<div class="clearfix"></div>-->
```

Comment:
It would be best if critical changes were only made to the `newsup-child` as to push changes to the website requires us to FTP files across. It would be a pain in the ass doing it for multiple themes. See if you can achieve this using only the child theme. (hint: Copy down the files into child from parent, and remove the lines of code there)

Reply:
oh right, my bad. gonna fix this now. thanks for reminding me. im not used to the child directory xD

Reply:
I doing this for Phaedra tbh. A production theme is a LOT to find differences in.

Reply:
yea, navigating all the built in functions and widgets kinda spooky too xD

but it slight manageable using the inspector tool on chrome and searching the entire project :( 

making do with it oui

Reply:
Have you seen the site using 1 alone and 2 alone? (Referring to how you numbered the title sections. Which one looks better to you? Just use whatever you think is best. I guessed 2 would look better but you should make that call.

Reply:
it looks much cleaner with just the one title IMO

![image](https://user-images.githubusercontent.com/38733594/78499620-b7d36f00-771f-11ea-9ac3-a48345603885.png)

im trying to reduce clutter, but not omit anything that we can potentially use. That extra title bar rel outtaz imo, but if you or asa thinks it belongs there, ill put it back.